content/post/static-analysis-for-games.md

```diff
@@ -92,10 +92,15 @@ Testing your game from end-to-end, as if you were a player, can be great for cat
 As a result, classical playtesting should often be considered as either a last resort, or as a "finishing pass" on a design you've already analyzed on it's own.
 This is particularly easy and effective if you have good tuning levers to work with, allowing your classical playtest to focus on nailing those values and improving game feel.
 
-## Conclusion
+## Every tool has a place
 
 In the end, classical playtesting can still be fun, informative, and highly effective, particularly for learning about subjective player experience, and performing holistic evaluation.
 But, as we hope we've convinced you in this blog post, it's only one tool in your toolbox.
 Use static analysis first to identify problems, hammer them out with focused tests and then use classical playtesting to refine feel and catch unforeseen issues.
 By doing this, you can catch mistakes early and guide your designs much more intentionally.
-As you sharpen these analysis skills, you'll eventually be able to accept, reject, and critique whole systems by inspection alone.
+
+As we've built games together, we've learned to **turn patterns into tools.**
```

Comment:
This new section is nice, but I don't feel like it fits. The conclusion isn't the place to be bringing in new information in my opinion. It should be a short summary of the takeaways, and maybe a call to action. I'm in favor of just cutting this and putting it in a different post, but I'm fine with just moving it to a different section. If you want to change the original ending line, you still have my blessing.

Reply:
Definitely agree on the call to action. Hmm...